Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Spawning/SpawnTicketInstance.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/RTTI/RTTI.h>
+#include <AzCore/RTTI/ReflectContext.h>
+#include <AzFramework/Spawnable/SpawnableEntitiesInterface.h>
+
+namespace ScriptCanvas::Nodeables::Spawning
+{
+    //! Wrapper around EntitySpawnTicket that can be used by ScriptCanvas
+    struct SpawnTicketInstance final
```

Comment:
Once https://github.com/o3de/o3de/pull/7924 and used here, this class we may want to look into removing this class and exposing the ticket directly to ScriptCanvas/Behavior Context if that's an option.